packages/truffle-config/README.md

```diff
@@ -1,2 +1,23 @@
 # truffle-config
 Utility for interacting with truffle-config.js files
+
+### Usage
+
+ ```javascript
+const TruffleConfig = require("truffle-config");
+
+// instantiate a default TruffleConfig object
```

Comment:
Instead of making these all part of one code block, let's make these h4's with separate code blocks!

Reply:
We'll want to shorten the wording to be more heading-appropriate (remove `a`/`the`, etc. and generally make each succinct)

Reply:
We could group some, too. Like `#### Config.detect()` and then have 1 code block with both examples.

Basically my goal here is that I want to be able to link to specific sections of the README easily.